x/wasm/handler.go

```diff
@@ -60,22 +60,23 @@ func handleStoreCode(ctx sdk.Context, k Keeper, msg *MsgStoreCode) (*sdk.Result,
 		return nil, err
 	}
 
-	codeID, err := k.Create(ctx, msg.Sender, msg.WASMByteCode, msg.Source, msg.Builder)
+	codeID, err := k.Create(ctx, msg.Sender, msg.WasmByteCode, msg.Source, msg.Builder)
 	if err != nil {
 		return nil, err
 	}
 
-	events := filterMessageEvents(ctx.EventManager())
```

Comment:
It seems you are missing the `filterMessageEvents`, which may be causing this other error.

Reply:
Yeah, this maybe the reason. Will modify this and see

Reply:
fixed in my last commits